spec/features/merchants/items/index_spec.rb

```diff
@@ -20,4 +20,24 @@
     expect(page).to_not have_content(@item_3.name)
     expect(page).to_not have_content(@item_4.name)
   end
+
+  it "Can enable and disable an item" do
```

Comment:
If we are disabling an item, is it supposed to be for one merchant or for all merchants? I think this currently disables for all merchants, but I'm not certain which way the user story is asking. I see it says to have a button next to an item's index page, but with the title being merchant items I'm wondering if they meant a merchant item index page?